### DISCUSSION TITLE
Not able to install properly on MAC

### DISCUSSION THREAD

**Discussion by vidhyasgr** [rancher-sandbox/rancher-desktop#1433] [Q&A] [answered]
I get the following error during Rancher Desktop installation.

<img width="1754" alt="Bildschirmfoto 2022-02-01 um 10 40 33" src="https://user-images.githubusercontent.com/9194684/151949842-166df4d8-ba6e-4c3d-abe7-55f3d7e01379.png">

**Comment by evertonlperes** [comment, marked as answer]
Hi @vidhyasgr 
I just saw your issue. Please, feel free to track it in there: https://github.com/rancher-sandbox/rancher-desktop/issues/1434
